### PR TITLE
Provides conan recipe file to build artifacts

### PR DESCRIPTION
**Pull request by deekshith-elear**
Adds conan recipe file to this repo

### CODE REVIEW

File: conanfile.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+    def build(self):
+        autotools = AutoToolsBuildEnvironment(self)
+        self.run("cd .. && autoreconf -fsi ")
+        autotools.configure(configure_dir="..",args=["--prefix=${PWD}"])
+        autotools.make()
+        autotools.install()
+    def package(self):
+        self.copy("*.h", dst="include", src="include")
+        self.copy("*", dst="lib", src="lib", keep_path=False)
```
Comment (manavkumarm) [inline conanfile.py:25]: Does this also copy header files?

**Review by manavkumarm — CHANGES_REQUESTED** [review]

File: conanfile.py (hunk 1 above)
Comment (deekshith-elear) [reply conanfile.py:25]: Yes, it copies `header files from source directory to package ` and ` libraries from build directory to lib directory in package`

File: conanfile.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
+from conans import ConanFile, AutoToolsBuildEnvironment, tools
```
Comment (manavkumarm) [inline conanfile.py:1]: README is missing. For repos that are built using auto tools, please include an instruction to run `make distclean` (to clean up files from previous builds) before running `/opt/bin/build_setup.sh`

**Review by manavkumarm — CHANGES_REQUESTED** [review]

File: conanfile.py (hunk 2 above)
Comment (deekshith-elear) [reply conanfile.py:1]: As per discussion I will address `make distclean` part in presentation and not modifying README for 3rd party libraries

File: conanfile.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+class JsonclibConan(ConanFile):
+    name = "json-c"
+    version = "0.1"
+    license = "<Put the package license here>"
+    author = "<Put your name here> <And your email here>"
+    url = "<Package recipe repository url here, for issues about the package>"
+    description = "<Description of jsonclib here>"
+    topics = ("<Put some tag here>", "<here>", "<and here>")
+    settings = "os", "compiler", "build_type", "arch"
+    options = { "shared": [True, False] }
```
Comment (manavkumarm) [inline conanfile.py:12]: Pending style fix: `[ True, False ]` - space after [, space before ]

**Review by manavkumarm — APPROVED** [review]

File: conanfile.py (hunk 3 above)
Comment (deekshith-elear) [reply conanfile.py:12]: handled

**Pull request merged**